src/modules/currencyWallets/api.js

```diff
@@ -390,7 +391,7 @@ export function makeCurrencyApi (
         }
       }
 
-      return getMax('0', balance)
+      return getMax('0', add(balance, '1'))
```

Comment:
Not sure why 1 was added here. Can you explain? Is this to compensate for the `lte` above?

Reply:
Yes, but only for [balance - 1, balance] case. My bad, fixed in next commit.